packages/core/content-type-builder/server/controllers/validation/types.js

```diff
@@ -115,6 +115,11 @@ const getTypeShape = (attribute, { modelType, attributes } = {}) => {
         maxLength: validators.maxLength,
       };
     }
+    case 'blocks': {
+      return {
+        required: validators.required,
+      };
+    }
```

Comment:
@remidej @Feranchz I just left required for now. I'm not sure how default is going to work exactly.